docs/section22/Rule22-9.md

```diff
@@ -0,0 +1,65 @@
+
+# CHW&CW - Rule 22-9  
+
+**Schema Version:** 0.0.10 **Mandatory Rule:** True
+**Rule ID:** 22-9  
+**Rule Description:** For Baseline chilled water system with cooling capacity of 300 tons or more, the secondary pump shall be modeled with a minimum flow of 25% of the design flow rate.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.3.10 Chilled-water pumps (System 7, 8, 11, 12 and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, 13, 7b, 8b, 11b, 12b, 13b.
+2. Primary and secondary CHW loops are modeled correctly in B-RMR.
+3. B-RMR chilled water system cooling capacity is 300 tons or more.
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. get_primary_secondary_loops()
+
+**Applicability Checks:**  
+
+- **Check 1:** Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): NEXT_APPLICABILITY_CHECK`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+- **Check 2:** Get primary and secondary loops for B-RMR: `primary_secondary_loop_dictionary = get_primary_secondary_loops(B_RMR, "COOLING")`
+
+  - Check if primary and secondary loops are modeled correctly, continue to rule logic: `if primary_secondary_loop_dictionary["PRIMARY"] AND primary_secondary_loop_dictionary["SECONDARY"]: CHECK_RULE_LOGIC`
```

Comment:
I think this RDS should do the following:
1. Use primary_secondary_loop_dictionaryIdentify to identify pairs of primary/secondary loop. (There should be just one such pair, but this is checked by 22-34, so it would flag any mistakes related to this.)
2. For each pair, (a) check that primary loop is served by chillers with capacity over 300 ton; if true then (b) check that secondary loop has min flow = 25%. If true, then pass.